pkg/cmd/cli/version/version.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017, 2019 the Velero contributors.
+Copyright 2017, 2019, 2020 the Velero contributors.
```

Comment:
This should actually be just 2020, if you wouldn't mind updating it.

I know many of the files are inconsistent with this, we're slowly updating them.